.gitignore

```diff
@@ -8,3 +8,4 @@
 /makelib/variables.mk
 /build-rkt*
 .vagrant
+tags
```

Comment:
If this file is only in the toplevel directory of the repository, then prepend it with `/`. If not, then ignore the comment.